extensions/bokeh_magic.py

```diff
@@ -27,6 +31,11 @@
 class BokehMagics(Magics):
     """Magic to embed Bokeh into the IPython notebook."""
 
+    if old_bokeh:
+        raise RuntimeError("We are deprecating the Bokeh magic. "
```

Comment:
It is no more chatty than with the `RuntimeError` but I agree `DeprecationWarning` is the proper exception to raise... I already pushed the change.